Include/codecs.h

```diff
@@ -27,6 +27,16 @@ PyAPI_FUNC(int) PyCodec_Register(
        PyObject *search_function
        );
 
+/* Unregister a codec search function.
+
+   Unregister a codec search function and clear the registry's cache.
+   If the search function is not registered, do nothing.
+   */
+
+PyAPI_FUNC(int) PyCodec_Unregister(
+       PyObject *search_function
+       );
```

Comment:
Please document it in Doc/c-api/codec.rst.

Reply:
oh, thanks, I forget to update it.